tests/smolyay_test.py

```diff
@@ -0,0 +1,200 @@
+import pytest
+import numpy
+from smolyay.smolyak import IndexGrid
+
+
+grid_points_expect = numpy.array([[0,0],[0,1],[0,2],[0,3],[0,4],[1,0],[1,1],[1,2],[2,0],[2,1],[2,2],[3,0],[4,0]], dtype=numpy.int32)
```

Comment:
Rather than make this a global variable, you can make it a test fixture:
```py
@pytest.fixture
def expected_points():
    return numpy.array(...)

def test_something(expected_points):
    assert point in expected_points
```